techniques/system/common/1.0/promises.st

```diff
@@ -790,17 +790,16 @@ bundle edit_line enforce_content(str)
 bundle edit_line fix_syslogd(syslogd)
 {
   delete_lines:
-    any::
-      "^(local6)\s+(?!${syslogd}).*"
-        comment => "Delete missconfigured rudder syslogd destination";
-      "^local6\.\*\s+${syslogd}"
-        comment => "Delete old rudder syslogd format";
+
+      "^\s*local6.*"
```

Comment:
That means we take over all local6 configuration of the user whatever it is.
Isn't it possible that they already use it for something else than rudder ?